### ISSUE TITLE
[Bug] Add button and Reorder button have different colors

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/theme-tabler#3]
When I go to the Categories CRUD, I notice the Add and Reorder buttons have different colors. In fact... everywhere... the Add button looks a bit "disabled": 

<img width="1511" alt="CleanShot 2023-03-23 at 11 44 32@2x" src="https://user-images.githubusercontent.com/1032474/227164347-dd4184dc-450c-46aa-a5cc-33875d2165f7.png">

That's because:
- Add is using `btn-primary`
- Reorder is using `btn-outline-primary`

But that's weird because... that doesn't look at all like the `btn-outline-primary` in tabler:
<img width="1512" alt="CleanShot 2023-03-23 at 11 47 14@2x" src="https://user-images.githubusercontent.com/1032474/227165092-e576e00b-bda7-4a25-8b17-dc0af5818274.png">

So here's what I'd do:
- I'd change the color of `btn-primary` to be more powerful; to pop; to basically look like the Reorder button looks now;
- I'd investigate why `btn-outline-primary` doesn't look like an outline button, and make it look like an outline button; 

Basically I'd make them look like the top buttons in Categories look now:
<img width="1512" alt="CleanShot 2023-03-23 at 11 48 35@2x" src="https://user-images.githubusercontent.com/1032474/227165555-07aca0f4-376c-4635-8dd5-b38891b8de2d.png">
